src/Indice.Psd2.Cryptography/Tokens/HttpMessageSigning/HttpSignatureDelegatingHandler.cs

```diff
@@ -100,6 +100,14 @@ HttpMessageHandler innerHandler
             return;
         }
 
+        /// <summary>
+        /// Change the default name of the <see cref="RequestSignatureCertificateHeaderName"/> and use a specific one
+        /// </summary>
+        /// <param name="name">The new name that will be used in signature certificate header</param>
+        public void SetRequestSignatureCertificateHeaderName(string name) {
```

Comment:
Same as with the response validation setting. I prefer it to be a plain old property. Properties after all are get and set methods over private fields.